src/styles/theme.ts

```diff
@@ -18,6 +18,7 @@ let theme = createMuiTheme({
     button: {
       textTransform: 'none',
     },
+    fontFamily: 'Wipro Akkurat TT, Wipro Akkurat Mono TT, Roboto, Helvetica, Arial, sans-serif',
```

Comment:
Is Roboto imported somewhere in the App? I don't think it's natively supported by all browsers except Chrome.
If I'm right it's a weak fallback.

Reply:
That's why it ends up with `Helvetica, Arial, sans-serif` :)